notification/twilio/validation.go

```diff
@@ -13,7 +13,9 @@ import (
 
 func validateRequest(req *http.Request) error {
 	if req.Method == "POST" {
-		req.ParseForm()
+		if err := req.ParseForm(); err != nil {
+			return errors.New("unable to parse form input")
+		}
```

Comment:
If Twilio sends an invalid payload, we should probably return a validation error (400). This will result in a 500 error

Reply:
That was my mistake; I'm used to needing to wrap with our validation package for the catch-all everywhere else.